README.md

```diff
@@ -1 +1,91 @@
-# discord-client
\ No newline at end of file
+# @progressive-victory/client
+
+`@progressive-victory/client` is a general use Discord bot client that extends [discord.js](https://github.com/discordjs/discord.js)
+
+## Usage
+
+Install of @progressive-victory/client:
+
+```sh
+npm install @progressive-victory/client
+yarn add @progressive-victory/client
+```
+
+The client has a builting interation handler. and does not need the `interactionCreate` event to be added. The Client Does **NOT** support refix commands
+
+## Files setup
+
+Example index.ts for a bot:
+
+```ts
+import { init } from '@i18n';
+import { Client } from '@progressive-victory/client';
+
+export const client = new Client({
+   receiveMessageComponents: true,
+   receiveModals: true,
+   receiveAutocomplete: true,
+   replyOnError: true,
+   splitCustomID: true,
+   splitCustomIDOn: '_',
+ useGuildCommands: false
+});
+
+(async function start() {
+    await client.init({
+        eventPath: './events',
+        buttonPath: './interactions/buttons',
+        selectMenuPath: './interactions/select_menus',
+        modalPath: './interactions/modals',
+        commandPath: './commands/chat/builders',
+        contextMenuPath: './commands/context_menu'
+    });
+
+    await client.login(process.env.TOKEN);
+
+    await client.deploy();
+});
+```
+
+### How to use event objects
+
+In a Dirctory path set in the `eventPath`
+Example event for Discord client event `ready`
+
+```ts
+import { Events } from 'discord.js';
+import { Event } from '@progressive-victory/client';
+
+export default new Event().setName(Events.ClientReady).setOnce(true).setExecute((client) => {
+    console.log(`Ready! Logged in as ${client.user.us}`);
```

Comment:
Also just noticed, "us"?

Reply:
Oh also I would suggest using the logger in the example maybe to discourage console.log